### DISCUSSION TITLE
how to dynamic reload   ssl key without break connection?

### DISCUSSION THREAD

**Discussion by calvin2021y** [uNetworking/uWebSockets#1668] [Q&A]
Is there a way to load ssl key like nginx  reload,  keep the connection open.

**Comment by uNetworkingAB** [comment]
No but you can change SSL certs for following connections without shutting down the server. If you look at SNI, there is a release post about it if you scroll through releases

**Reply by uNetworkingAB** [reply]
Combining it with maxLifetime and you will eventually migrate all connections
